CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 ## 1.2.0-SNAPSHOT
 
 * Fix help-for for some restructure methods [#275](https://github.com/metosin/compojure-api/pull/275) by [Nicolás Berger](https://github.com/nberger)
+* In `compojure.api.swagger`, the `swagger-ui` and `swagger-docs` now take options map with `uri` key instead of separate optional path & vararg opts.
```

Comment:
Actually, the old consumed varargs, so there was a common 1-arity version. Now with assertion for the parameter type.